specification/datadog/Datadog.Management/DatadogMonitorResource.tsp

```diff
@@ -0,0 +1,192 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+namespace Microsoft.Datadog;
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+#suppress "@azure-tools/typespec-azure-core/no-private-usage" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@Azure.ResourceManager.Private.armResourceInternal(IdentityProperties)
+@TypeSpec.Http.Private.includeInapplicableMetadataInPayload(false)
+model DatadogMonitorResource
+  extends Azure.ResourceManager.Foundations.TrackedResource {
+  ...ResourceNameParameter<
+    Resource = DatadogMonitorResource,
+    KeyName = "monitorName",
+    SegmentName = "monitors",
+    NamePattern = ""
+  >;
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-invalid-envelope-property" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  sku?: ResourceSku;
+
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-invalid-envelope-property" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @Azure.ResourceManager.Private.armResourcePropertiesOptionality(true)
+  properties?: MonitorProperties;
+
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-invalid-envelope-property" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  identity?: IdentityProperties;
+}
+
+@armResourceOperations
+interface DatadogMonitorResourcesOps
+  extends Azure.ResourceManager.Legacy.LegacyOperations<
+      {
+        ...ApiVersionParameter,
+        ...SubscriptionIdParameter,
+        ...ResourceGroupParameter,
+        ...Azure.ResourceManager.Legacy.Provider,
+
+        /**
+         * Monitor resource name
+         */
+        @path
+        @segment("monitors")
```

Comment:
This should be the default segment for a `Monitor` resource, no?

Reply:
It is expected to have this segment part in a legacy operation